Makefile

```diff
@@ -22,17 +22,17 @@ fmt:
 	go fmt ./...
 
 .PHONY: build
-build: deps fmt $(bin)
+build: fmt $(bin)
 
 $(bin):
-	go build
+	CGO_ENABLED=0 go build -o ./$(bin)
 
 .PHONY: test
 test: test-unit test-integration
 
 .PHONY: test-unit
 test-unit:
-	go test $(TESTS) -run 'Unit'
+	go test $(TESTS) -run 'Unit' -coverprofile=coverage.out
```

Comment:
Some of these changes don't match the styleguides.
Do the styleguides need updating?
https://github.com/companieshouse/styleguides/blob/master/Makefiles/go_makefiles.md

Reply:
Hadn't seen that, I'll revert the changes. 

`-coverprofile=coverage.out` displays unit test coverage for each file it's run against, so I think that should stay as it's useful to see. I'd suggest the styleguide would need updating with that. 

I also get an error when leaving `deps` in the build command on line 28 so decided to take it out, I can't seem to find it in any other go repos we use